auth.go

```diff
@@ -235,34 +235,12 @@ func createDB(config *authaus.Config) (success bool) {
 }
 
 func loadOrCreateGroup(icentral *imqsauth.ImqsCentral, groupName string, createIfNotExist bool) (*authaus.AuthGroup, error) {
-	if existing, eget := icentral.Central.GetRoleGroupDB().GetByName(groupName); eget == nil {
-		return existing, nil
-	} else if strings.Index(eget.Error(), authaus.ErrGroupNotExist.Error()) == 0 {
-		if createIfNotExist {
-			group := &authaus.AuthGroup{}
-			group.Name = groupName
-			if ecreate := icentral.Central.GetRoleGroupDB().InsertGroup(group); ecreate == nil {
-				fmt.Printf("Group %v created\n", groupName)
-				return group, nil
-			} else {
-				fmt.Printf("Error inserting group %v: %v\n", groupName, ecreate)
-				return nil, ecreate
-			}
-		} else {
-			return nil, eget
-		}
+	if group, error := authaus.LoadOrCreateGroup(icentral.Central, groupName, createIfNotExist); error == nil {
+		fmt.Printf("Group %v created\n", groupName)
+		return group, nil
 	} else {
-		return nil, eget
-	}
-}
-
-func saveGroup(icentral *imqsauth.ImqsCentral, group *authaus.AuthGroup) bool {
```

Comment:
I think this function just has to live here - since it exists solely for the command-line interface